src/web/app/components/error-report/error-report.component.spec.ts

```diff
@@ -14,17 +17,108 @@ describe('ErrorReportComponent', () => {
         FormsModule,
         HttpClientTestingModule,
       ],
+      providers: [HttpRequestService],
     })
     .compileComponents();
   }));
 
   beforeEach(() => {
     fixture = TestBed.createComponent(ErrorReportComponent);
     component = fixture.componentInstance;
+    httpRequestService = TestBed.get(HttpRequestService);
     fixture.detectChanges();
   });
 
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should have "Something went wrong" header', () => {
```

Comment:
I think one snapshot testing can cover the following scenarios?
- 'should have "Something went wrong" header'
- 'should display error message'
- 'should have the requestID form disabled '
- 'should have pre-filled Subject input form'
- 'should snap with default view'
- 'should snap with some content'

Reply:
Roger. Removed redundant test cases